assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -556,8 +572,24 @@ $cart-item-label-offset:                $cart-thumbnail-maxWidth + $cart-item-sp
 
 .previewCartItem-image {
     @include grid-column(4, $float: false);
-    padding: spacing("single");
+    padding: 0;
     text-align: center;
+    position: relative;
+
+    img {
```

Comment:
Is this a good candidate for a `@mixin` since it is used in a couple of places.

Reply:
Will do